apps/extension/src/routes/popup/login.tsx

```diff
@@ -0,0 +1,42 @@
+import { Button } from 'ui/components';
+import { FadeTransition, LoginForm } from '../../components';
+import { PagePath } from '../page/paths';
+
+export const Login = () => {
```

Comment:
We should think about the design of this page later. It is missing a kind of branding element.